### PR TITLE
[DOCS] Add `:` to render multiple inline macros in Asciidoctor

### PR DESCRIPTION
**Pull request by jrodewig**
Without some breaking text, Asciidoctor cannot render two consecutive inline macros without an `:` character. This adds that character so both AsciiDoc and AsciiDoctor render the macros as intended.

Relates to /elastic/docs#827

### Asciidoctor before changes
Screen Shot 2019-04-26 at 5 37 11 PM

### Asciidoctor after changes
Screen Shot 2019-04-26 at 5 35 08 PM

### AsciiDoc after changes
Screen Shot 2019-04-26 at 5 34 49 PM

### CODE REVIEW

File: docs/reference/settings/notification-settings.asciidoc [hunk 1]
```diff
@@ -35,8 +35,10 @@ required. For more information, see
 {xpack-ref}/encrypting-data.html[Encrypting sensitive data in {watcher}].
 
 `xpack.watcher.history.cleaner_service.enabled`::
-added[6.3.0,Default changed to `true`.]
-deprecated[7.0.0,Watcher history indices are now managed by the `watch-history-ilm-policy` ILM policy]
+// {empty} attributes are included so both macros render as intended.
```
Comment (nik9000) [inline docs/reference/settings/notification-settings.asciidoc]: I think I'd prefer to use `ifdef::asciidoctor[]` blocks to handle the different way to write this. I think this way works but we only need it if we're going to support both Asciidoctor and AsciiDoc with the same source exactly. I think having `ifdef::asciidoctor[]` will let us search for it after the migration and we can remove all of the usages. Though we could also remove all the usages of `{empty}`. Either way is fine, but I think `ifdef::asciidoctor[]` is a little more consistent.

Comment (jrodewig) [reply docs/reference/settings/notification-settings.asciidoc]: The `{empty}` usage is needed to get Asciidoctor to render correctly, not AsciiDoc. I'm not sure we can remove it after migration, even if we implement `ifdef::asciidoctor[]`.

It seems Asciidoctor won't render two consecutive inline macros without any text between. The `{empty}` attribute is a hacky way around that.

I'm happy to implement `ifdef::asciidoctor[]` if we find a better way of rendering the two inline macros in Asciidoctor. My research hasn't turned up any alternative solutions though.

Comment (nik9000) [reply docs/reference/settings/notification-settings.asciidoc]: Does this work for Ascidoctor?
```
added:[6.3.0,Default changed to `true`.]
deprecated:[7.0.0,Watcher history indices ...]
```

Comment (jrodewig) [reply docs/reference/settings/notification-settings.asciidoc]: Unfortunately not.

**Input**
```
added[6.3.0,Default changed to `true`.]
deprecated[7.0.0,Watcher history indices are now managed by the `watch-history-ilm-policy` ILM policy] 
```
**Asciidoctor output**
Screen Shot 2019-05-01 at 1 50 34 PM

Comment (nik9000) [reply docs/reference/settings/notification-settings.asciidoc]: With the single `:`s between the name and the `[`. That is asciidoctor's native syntax. Nothing between the name and the `[` is a thing we twist into place based on heuristics that don't work here.

Comment (jrodewig) [reply docs/reference/settings/notification-settings.asciidoc]: :doh: That did the trick. Fixed it up with f31252b. Thanks again for your near-infinite patience and help, @nik9000.

Comment (jrodewig) [reply docs/reference/settings/notification-settings.asciidoc]: Pushed f7ba3b0 to implement `ifdef::asciidoctor[]`. Thanks again, Nik!

**Review by nik9000 — APPROVED** [review]
LGTM!

This'll give us an obvious thing to clean up once we don't have to build with AsciiDoc any more!

**Pull request merged**